pkg/common/tpm_attestor.go

```diff
@@ -65,6 +64,6 @@ func GetPubHash(cert *x509.Certificate) (string, error) {
 	pubKey := cert.PublicKey.(*rsa.PublicKey)
 	pubBytes := x509.MarshalPKCS1PublicKey(pubKey)
 	pubHash := sha256.Sum256(pubBytes)
-	hashEncoded := base64.StdEncoding.EncodeToString(pubHash[:])
+	hashEncoded := fmt.Sprintf("%x", pubHash)
```

Comment:
is `hashEncoded` now a misleading name?

Reply:
I think it is fine, it is still the shasum, but hex encoded, not base64 encoded.